section2/reflection.md

```diff
@@ -1,15 +1,40 @@
 ## Section 2 Reflection
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
+```some people think it's all nature and no nurture, but if you want to grow you can grow.  
+learning can come in all shapes and sizes, and failing / retrying things, is a good way to learn
+```
 
 2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
+```
+I like to experiment.  I don't mind getting things wrong.
+One way that I do not is that if I "think" I get a concept, i tend to half ass the rest of the problem.  learning by repititon is a powerful thing.
```

Comment:
Ha, fair. And I agree, some repetition is good. Probably something to keep your eye on when collaborating - if you veer towards this but your partner really needs that practice, another reason to push yourself to not half ass, you know?